src/locales/cy/pages.yml

```diff
@@ -1,10 +1,11 @@
 intro:
   h1: Intro
+shared:
+  abandon: "<a href='/kbv/abandon' class='govuk-link' data-id='abandon'>Mae angen i mi brofi fy hunaniaeth mewn ffordd arall</a>"
 rti-payslip-national-insurance:
-  title: Enter the amount of National Insurance you paid from a recent payslip
-  inset: Use a payslip dated after {{ threeMonthsAgo }} to answer this question
+  title: Rhowch swm yr Yswiriant Gwladol a dalwyd gennych o slip cyflog diweddar
```

Comment:
We may need to add a separate title and h1 here, just awaiting confirmation from UCD

Reply:
Had confirmation we don't need to (title and h1 are the same)